commands/command_migrate_import.go

```diff
@@ -29,6 +30,67 @@ func migrateImportCommand(cmd *cobra.Command, args []string) {
 	}
 	defer db.Close()
 
+	if migrateNoRewrite {
+		if len(args) == 0 {
+			ExitWithError(errors.Errorf("fatal: the migrate import command requires a list of files to import when using the --no-rewrite flag"))
+		}
+
+		ref, err := git.CurrentRef()
+		if err != nil {
+			ExitWithError(errors.Wrap(err, "fatal: unable to find current reference"))
+		}
+
+		sha, _ := hex.DecodeString(ref.Sha)
```

Comment:
Great; throwing this error away is reasonable since we trust `git.CurrentRef()` to always give back a valid SHA-256 `ref.Sha`.